src/main/java/science/icebreaker/account/AccountService.java

```diff
@@ -158,4 +163,58 @@ private String generateJwtToken(Account account) {
                 .signWith(SignatureAlgorithm.HS512, jwtSecret)
                 .compact();
     }
+
+    public void confirmAccount(String confirmationToken) {
+        AccountConfirmation accountConfirmation = accountConfirmationRepository
+                .findAccountConfirmationByConfirmationToken(confirmationToken);
+
+        Date oneHourAgo = new Date(System.currentTimeMillis() - 3600 * 1000);
```

Comment:
This should be tweakable and in general, available on a wider scope in case we add a cron job for cleanup for example

Reply:
you mean expire time of confirmation token?

Reply:
yes

Reply:
Yeah, this line can be safely deleted and token will not expire or configured from properties file by taking out variable of expire time, in this case 3600.

Reply:
This can be left out for now as well I guess